packages/transactions/src/coders/receipt.ts

```diff
@@ -26,8 +26,12 @@ export enum ReceiptType /* u8 */ {
 
 export type ReceiptCall = {
   type: ReceiptType.Call;
-  /** Contract ID of current context if in an internal context, zero otherwise (b256) */
+  /**
+   * @deprecated This property is deprecated and it will be removed soon. Use property `id` instead.
```

Comment:
I suppose that it may be a lot of friction for consumers who are currently using this field to decode receipt logs, I would be in favour of deprecation here as that tends to be a core function of this type.